.github/workflows/test.yml

```diff
@@ -45,7 +45,6 @@ jobs:
       fail-fast: false
       matrix:
         python:
-          - "3.10"
```

Comment:
Note: We ultimately want to add Python 3.12 support, which is why I want to leave the matrix logic as is despite us only testing with 3.11 right now.